parm/aqm.rc

```diff
@@ -29,9 +29,9 @@ ctm_pmdiag: true
 #
 #  Input emissions
 #
-emission_sources: anthro fire
+emission_sources: anthro fire pt
```

Comment:
@bbakernoaa, I think `pt` is not necessary here in this PR. This causes a failure of run_fcst.

Reply:
I'll be off from tomorrow. I'll merge this PR and remove `pt` on my end